scripts/setup/debians/cmake/changelog

```diff
@@ -0,0 +1,16 @@
+cmake (3.16.3-1ubuntu1+focal1) focal; urgency=medium
+
+  * Force 64bit offsets
+    - This fixes a cmake bug on 32bit systems, allowing compilation
```

Comment:
does anyone still use 32 bit systems?

Reply:
armhf is basically a 32bit system. We don't need it for amd64 but I actually compiled and added a cmake debian from this as well to avoid some conflicts with cmake-data. APT is not as smart as it should be.